docs/remote_inference_blueprints/cohere_connector_embedding_blueprint.md

```diff
@@ -24,7 +24,9 @@ POST /_plugins/_ml/connectors/_create
       "headers": {
         "Authorization": "Bearer ${credential.cohere_key}"
       },
-      "request_body": "{ \"texts\": ${parameters.texts}, \"truncate\": \"${parameters.truncate}\", \"model\": \"${parameters.model}\" }"
+      "request_body": "{ \"texts\": ${parameters.prompt}, \"truncate\": \"${parameters.truncate}\", \"model\": \"${parameters.model}\" }", 
```

Comment:
@dtaivpp This PR already fixed the input filed name https://github.com/opensearch-project/ml-commons/pull/1274, We don't need to change `parameters.texts` to `parameters.prompt` with that PR

Can you rebase and update this PR?